### DISCUSSION TITLE
A few questions

### DISCUSSION THREAD

**Discussion by ElisendaPeris** [Mattk70/Chirpity-Electron#72] [General]
Hello,

I am new using Chirpity and I've been comparing it to the BirdNET-analyser. I really like that Chirpity is way more user friendly, and allows to directly listen to the audio fragments where a detection is made, in fact, in general I think it is way easier to use. Nevertheless I have a few questions that I haven't been able to find:

1. When analysing the recordings, does it just analyse the 3s fragments as in the default options BirdNet, or a short audio overlap is included to be sure that all calls will be detected (as allowed in BirdNet)?
2. Is it possible to select which version of BirdNet Chirpity uses? I've heard that older versions of birdnet are better for owls and I would like to run my analysis with those versions, would it be possible to select which BirdNet version to use in Chirpity?
3. I've been having problems when editing the detected calls (changing the species, adding comments and call counts). When saving, I'm not able to find this modifications within the program and I don't  see this changes when downloading the .csv data. I would like to know if I'm doing something wrong or if this is a general issue.

Thank you very much for your imputs and for developing such an interface!

Cheers,

**Comment by Mattk70** [comment]
Hi @ElisendaPeris, thanks for your comments! About your questions:

> When analysing the recordings, does it just analyse the 3s fragments as in the default options BirdNet, or a short audio overlap is included to be sure that all calls will be detected (as allowed in BirdNet)?

It's the 3s fragments (windows)  with no overlap, as with the default BirdNET settings. In my experience, having a call span two 3s windows hasn't been a particular issue for identifying calls. However, there is an issue where a call chopped off at the beginning or end of a window is more likely to generate a false positive detection. Since the overlap adds more window edges, you end up with more false positives. It also takes longer to analyse. That's been my experience, anyway.

> Is it possible to select which version of BirdNet Chirpity uses? I've heard that older versions of birdnet are better for owls and I would like to run my analysis with those versions, would it be possible to select which BirdNet version to use in Chirpity?

No, it's just the 2.4 model, I'm afraid. It is generally better than the older versions, although as you say, there may be some species where the performance has deteriorated. 

> I've been having problems when editing the detected calls (changing the species, adding comments and call counts). When saving, I'm not able to find this modifications within the program and I don't see this changes when downloading the .csv data. I would like to know if I'm doing something wrong or if this is a general issue.

I'm sorry to hear that. It's not a issue I've heard from anyone before, and it works for me, Perhaps you could go into a little more detail - if you follow the guidance in the issue / bug report template, this will help me pin things down.
